examples/habana/streaming_llm/run_yi_34b_fp8_streaming.sh

```diff
@@ -0,0 +1,25 @@
+echo "========== START TO MEASURE =========="
+QUANT_CONFIG=../quantization_config/maxabs_measure.json python run_streaming_llm.py \
+    --model_name_or_path=01-ai/Yi-34B-Chat \
```

Comment:
shall be arguments passed by the users

Reply:
It can if they use `run_streaming_llm.py`. I write these shell scripts here just for OOB and a quick try.